recipes/widgetsnbextension/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0.0" %}
+
+package:
+    name: widgetsnbextension
+    version: {{ version }}
+
+source:
+    fn: widgetsnbextension-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/41/98/e2424df4a1222df72451bddc4471ec15482cc8c125203ffcf1923173e6a6/widgetsnbextension-{{ version }}.tar.gz
```

Comment:
Well, they decided awhile ago that they don't want to support the old PyPI because they are trying to make Warehouse. However, the net result is we get sporadic issues with PyPI and breaks like this one. Only after lots of complaining does the situation get improved a little.